bundles/org.openhab.binding.tado/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="tado:zone">
+		<instruction-set targetVersion="1">
+			<update-channel id="heatingPower">
+				<type>Number:Dimensionless</type>
```

Comment:
```suggestion
				<type>tado:heatingPower</type>
```

Reply:
Oh. I see. The doc is not very clear..

Reply:
It's not exactly intuitive. Therefore I would also highly recommend to test the migration, since even a small mistake (like this) can break the upgrade.

Reply:
@andrewfg - just to be sure, did you see the comment above? Please verify that migration is now tested and working.